packages/web-runtime/themes/owncloud/theme.json

```diff
@@ -5,12 +5,14 @@
     "logo": "themes/owncloud/assets/logo.svg"
   },
   "ios": {},
-  "web": {
-    "default": {
+  "web": [
+    {
       "general": {
         "name": "ownCloud",
+        "title": "Light Theme",
         "slogan": "ownCloud – A safe home for all your data",
         "privacyUrl": "",
+        "isDark": false,
```

Comment:
can we name it darkMode or isDarkMode? that sounds better to me and gives more context

Reply:
Thinking about it again, it seems a bit odd because the theme *isn't* dark mode .. it is used *for* dark mode. So to me isDark seems actually more appropriate.

Reply:
Yeah, not to be confused with e.g. https://vueuse.org/core/usePreferredDark 's `isDark`... - my `"isDark"` attribute should note whether a theme is created to be "used with dark mode". We could change it to `isDarkModeTheme`? Open for suggestions!